game/PsyDoom/Config/Config.cpp

```diff
@@ -119,6 +119,16 @@ CheatKeySequence gCheatKeys_XRayVision;
 CheatKeySequence gCheatKeys_VramViewer;
 CheatKeySequence gCheatKeys_NoTarget;
 
+//------------------------------------------------------------------------------------------------------------------------------------------
+// Multiplayer settings
+//------------------------------------------------------------------------------------------------------------------------------------------
+bool        gbNoFriendlyFire;
+bool        gbExitDisabled;
+int32_t     gFragLimit;
+int32_t     gPreserveAmmoFactor;
+bool        gbPreserveKeys;
+bool        gbMPThings;
+
```

Comment:
We should probably name this variable a little more descriptively so it's more obvious what it does at a glance. Maybe 'gbForceSpawnMpThings' or something like that?

Reply:
https://github.com/ts-miller/PsyDoom/commit/3d000d45e1916810a059afdc46c001f80cce3a35